.github/workflows/main.yaml

```diff
@@ -7,13 +7,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set up JDK 8
+      - name: Set up JDK 17
```

Comment:
Is Java 17 required for the license check? I would better keep the build on java 8 to make sure we are fully compatible

Reply:
Yes, to run the recent plugins, JDK 8 is too old.

As far as I know, more recent versions of the JDK can still cross-compile to version 8 as long as the sources are also not older as version 8.

Reply:
Alternatively we could have a separate `<profile>` that adds this plugin and is run in a separate build action.

Reply:
Will do